perception/lidar_centerpoint/config/detection_class_adapter.param.yaml

```diff
@@ -0,0 +1,38 @@
+/**:
+  ros__parameters:
+    allow_remapping_by_area_matrix:
+      # NOTE(kl): We turn all vehicles into trailers if they go over 3x12 m2.
+      # NOTE(kl): We turn cars into trucks if they have an area between 2.2 x 5.5 and 2.5 * 7.9 m2
+      # row: original class. column: class to remap to
+      #UNKNOWN, CAR, TRUCK, BUS,  TRAILER, MOTORBIKE, BICYCLE,PEDESTRIAN
+      [0,       0,   0,     0,    0,       0,         0,      0,         #UNKNOWN
+       0,       0,   1,     1,    1,       0,         0,      0,         #CAR
+       0,       0,   0,     0,    1,       0,         0,      0,         #TRUCK
+       0,       0,   0,     0,    1,       0,         0,      0,         #BUS
+       0,       0,   0,     0,    0,       0,         0,      0,         #TRAILER
+       0,       0,   0,     0,    0,       0,         0,      0,         #MOTORBIKE
+       0,       0,   0,     0,    0,       0,         0,      0,         #BICYCLE
+       0,       0,   0,     0,    0,       0,         0,      0]         #PEDESTRIAN
+
+    min_area_matrix:
+      #UNKNOWN, CAR,   TRUCK,   BUS,   TRAILER,  MOTORBIKE, BICYCLE, PEDESTRIAN
+      [ 0.000,  0.000,  0.000,  0.000,  0.000,   0.000,     0.000,   0.000,      #UNKNOWN
+        0.000,  0.000, 12.100, 19.750, 36.000,   0.000,     0.000,   0.000,      #CAR
+        0.000,  0.000,  0.000,  0.000, 36.000,   0.000,     0.000,   0.000,      #TRUCK
+        0.000,  0.000,  0.000,  0.000, 36.000,   0.000,     0.000,   0.000,      #BUS
+        0.000,  0.000,  0.000,  0.000,  0.000,   0.000,     0.000,   0.000,      #TRAILER
+        0.000,  0.000,  0.000,  0.000,  0.000,   0.000,     0.000,   0.000,      #MOTORBIKE
+        0.000,  0.000,  0.000,  0.000,  0.000,   0.000,     0.000,   0.000,      #BICYCLE
+        0.000,  0.000,  0.000,  0.000,  0.000,   0.000,     0.000,   0.000]      #PEDESTRIAN
+
+
+    max_area_matrix:
+      #UNKNOWN, CAR,   TRUCK,  BUS,    TRAILER,   MOTORBIKE, BICYCLE, PEDESTRIAN
+      [ 0.000,  0.000,  0.000,  0.000,   0.000,   0.000,     0.000,   0.000,      #UNKNOWN
+        0.000,  0.000, 19.750, 36.000,     inf,   0.000,     0.000,   0.000,      #CAR
```

Comment:
Cloud you update the values since the max-area params of multi-object-tracker have updated in https://github.com/autowarefoundation/autoware.universe/pull/1710.

Reply:
Could you elaborate on which parameters do you refer?

From #1710 I understood the following:
 > I think it is better to make a trailer for anything over 12m in post-process of CenterPoint and apollo instance segmentation.
>Also, about width, 2.5m maybe too strict, it seems to be good to 3.0m

For which reason, the `TRAILER` column of `min_area_matrix` is 36.0 and the `TRAILER` column of `max_area_matrix` is `inf`. In this way, all `car`, `truck`, and `bus` instances become `trailer` when having an area over 36.0 

Are you asking me to make `cars` into 'trucks' for the range [12.10, 36.0] and `cars` into `buses` for the range [36.0, 40.0] ? In that case there would be an inconsistency with the previous quotation

Reply:
@knzo25 
I see. I don't want to use `19.750` since this value cannot be found anywhere in the latest main now. How about avoiding to map CAR to BUS?
car → truck: 12.1. <= area <= 36.0
car → trailer: 36.0 < area

Reply:
Done!